docs/cookbooks/common_workflows.md

```diff
@@ -86,31 +86,53 @@ SELECT
   -- format:on
 ```
 
-## Add a new field to clients_daily
-
-Adding a new field to `clients_daily` also means that field has to propagate to several
-downstream tables, so this is a more complex case.
-
-1. Open the `clients_daily_v6` `query.sql` file and add new field definitions
-1. Run `./bqetl query validate telemetry_derived.clients_daily_v6` to dry run and format the query
-1. Run `./bqetl query schema update telemetry_derived.clients_daily_v6 --update_downstream` to make local `schema.yaml` updates and update schemas of downstream dependencies
-    * This requires GCP access
-    * `schema.yaml` files of downstream dependencies, like `clients_last_seen_v1`, will be automatically updated
-1. Open PR with changes
-    * The `dry-run-sql` task is expected to fail at this point due to mismatch with deployed schemas
-1. PR reviewed and approved
+## Add a new field to a table schema
```

Comment:
This section is intended to be for `clients_daily` specifically, because it's a common but particularly complex case. In particular the `--update-downstream` and `--force` options discussed here are not necessary in the general case.

I'd be happy to see a separate simpler section for how to add a new field to a single table, but I think there's still merit to having a specific cookbook for desktop `clients_daily`.

Reply:
I had not fully absorbed the changed here when writing the above. Your changes here match the spirit of what I was suggesting 👍

Reply:
Understood @jklukas. However... would you say is more accurate if I highlight that  `--update-downstream` and `--force` optional in general, but mandatory for clients_daily and any other table that has dependencies?

Reply:
`--force` should only be used in very specific cases (`clients_last_seen`-type tables) as it skips some checks that would otherwise catch some error scenarios. `--update_downstream` will take much longer to complete, so probably is also not something we should recommend by default; include it only if you know there are downstream dependencies whose schemas will need to change.

Reply:
Thanks Jeff, added your comments.